### ISSUE TITLE
Null Object Reference crash near game start

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#703]
Win7x64, 0.5.2 full.

![dq-start](https://cloud.githubusercontent.com/assets/16719964/14824272/b2453460-0bd5-11e6-92e8-7b45e87c13d4.png)

```
ndows)>FlxUIButton name = movies_btn
FlxUIButton name = music_btn
FlxUIButton name = sfx_btn
FlxUIButton name = party_btn
FlxUIButton name = btn_journal
FlxUIButton name = btn_town
FlxUIButton name = tips_btn
FlxUIButton name = btn_battle
FlxUIButton name = exit_btn
FlxUIButton name = options_btn
FlxUIButton name = menu_btn
FlxUIButton name = bonus_btn
FlxUIButton name = movies_btn
FlxUIButton name = music_btn
FlxUIButton name = sfx_btn
FlxUIButton name = party_btn
FlxUIButton name = btn_journal
FlxUIButton name = btn_town
FlxUIButton name = tips_btn
FlxUIButton name = btn_battle
FlxUIButton name = exit_btn
FlxUIButton name = options_btn
FlxUIButton name = menu_btn
FlxUIButton name = bonus_btn
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 222
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/Na
tiveApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_back
end/native/NativeApplication.hx line 368
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 145
Called from *::_Function_2_1 lime/app/Application.hx line 110
Called from lime.app.Application::render lime/app/Application.hx line 762
Called from openfl.display.Stage::render openfl/display/Stage.hx line 677
Called from openfl.display.DisplayObjectContainer::__broadcast openfl/display/Di
splayObjectContainer.hx line 355
Called from openfl.display.DisplayObject::__broadcast openfl/display/DisplayObje
ct.hx line 251
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDi
spatcher.hx line 237
Called from flixel.FlxGame::onEnterFrame flixel/FlxGame.hx line 529
Called from flixel.FlxGame::step flixel/FlxGame.hx line 662
Called from com.leveluplabs.tdrpg.Main::update com/leveluplabs/tdrpg/Main.hx lin
e 1462
Called from flixel.FlxGame::update flixel/FlxGame.hx line 751
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 178
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 168
Called from com.leveluplabs.tdrpg.Popup_MapPreview::update com/leveluplabs/tdrpg
/Popup_MapPreview.hx line 108
Called from com.leveluplabs.tdrpg.TDRPGPopUp::update com/leveluplabs/tdrpg/TDRPG
PopUp.hx line 193
Called from flixel.input.actions.FlxActionAnalog::update flixel/input/actions/Fl
xAction.hx line 113
Called from flixel.input.actions.FlxAction::update flixel/input/actions/FlxActio
n.hx line 250
Called from flixel.input.actions.FlxActionAnalog::check flixel/input/actions/Flx
Action.hx line 132
Called from com.leveluplabs.tdrpg.Popup_MapPreview::onActionAnalog com/levelupla
bs/tdrpg/Popup_MapPreview.hx line 632
AL lib: (EE) alc_cleanup: 1 device not closed
```

**Comment by IBwWG** [comment]
Doing it a second time, it didn't seem to happen until I moused over the title of the scroll.  I think it may be the same as or related to #691.

**Comment by IBwWG** [comment]
Aha!  I think I have it.  So, whether I start fresh or just use the 0-level save that's left behind after crashing, it's the same: click BATTLE, and then you can sit there as long as you want without crashing, but as soon as you mouse out of BATTLE, crash.

**Comment by IBwWG** [comment]
In fact, this is true of #691 too, except that I didn't even have to mouse out of the ??? button entirely--simply moving the mouse a bit within the button crashed it.

**Comment by IBwWG** [comment]
Ah, which, more precisely, is also true about this start-game bug too.  So it's mousemove, not mouseout, on the map button, that's causing it.

**Comment by IBwWG** [comment]
It's odd though, because I managed to fight a battle...

**Comment by larsiusprime** [comment]
Fixed in 0.5.3

**Comment by IBwWG** [comment]
Yep, working here on 0.5.3.  Thanks!
